SWADroid/src/main/res/xml/tables.xml

```diff
@@ -385,6 +385,134 @@
         obligatory="true"
         type="foreign-key" />
     </table>
+    <table
+        name="games"
+        new-in-version="19"
```

Comment:
When new tables are added to the data model it is necessary to increase the version of the model to inform the database that the model has changed and must be updated.

The version of the data model is defined in **line 4** of this file and should look like this:

`version="22" >`

The new tables must indicate from which version of the model they first appear in order for the update process to run correctly.

```suggestion
        new-in-version="22"
```